src/llfs/committable_page_cache_job.hpp

```diff
@@ -206,12 +206,15 @@ class CommittablePageCacheJob
 
   Status drop_deleted_pages(u64 callers);
 
+  BoxedSeq<PageId> finalized_deleted_page_ids() const;
+
   //+++++++++++-+-+--+----- --- -- -  -  -   -
 
   std::shared_ptr<const PageCacheJob> job_;
   boost::intrusive_ptr<FinalizedJobTracker> tracker_;
   PageRefCountUpdates ref_count_updates_;
   std::unique_ptr<WriteNewPagesContext> write_new_pages_context_;
+  std::unordered_set<PageId, PageId::Hash> finalized_deleted_pages_;
```

Comment:
Suggest rename to `not_found_deleted_pages_` or `already_dropped_deleted_pages_` and adjust the rest of the code accordingly.

Reply:
Fixed, and changed `parallel_drop_pages`.